app/src/container.js

```diff
@@ -2,34 +2,50 @@
 
 "use strict";
 
+/**
+ * Application's service container, here we register all the services used by
+ * the application.
+ */
+
+require("dotenv").load();
```

Comment:
what does this do?

Reply:
Without this the environment variables loaded by dotenv aren't available in this file

Reply:
great, that should be the comment for that statement ;)